### ISSUE TITLE
Inception V3 checkpoint in slim missing variables

### ISSUE THREAD

**Issue by AshishBora** [tensorflow/models#422]
In the table given [here](https://github.com/tensorflow/models/tree/master/slim), Inception V3 checkpoint link points to [this](http://download.tensorflow.org/models/inception_v3_2016_08_28.tar.gz). When used to load the model it complains:

`Tensor name "InceptionV3/Mixed_5b/Branch_2/Conv2d_0c_3x3/biases" not found in checkpoint.`

I tried to run [inspect_checkpoint] (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/inspect_checkpoint.py) and verified that `InceptionV3/Mixed_5b/Branch_2/Conv2d_0c_3x3/biases` is not in the list of outputs. Interestingly, `InceptionV3/Mixed_5b/Branch_2/Conv2d_0c_3x3/weights` does exist in the list of outputs. Quite odd to have the weights but not the biases in there.

Another thing I noticed is that the checkpoint is only 100 MB in size as compared to one downloaded using 
`curl -O http://download.tensorflow.org/models/image/imagenet/inception-v3-2016-03-01.tar.gz`.
as suggested on [this](https://github.com/tensorflow/models/tree/master/inception) page.

Maybe the model checkpoint link is wrong?

**Comment by AshishBora** [comment]
Oops. Realized that biases have been removed from the checkpoint because of BatchNorm.

These were present in the model I created because I wasn't using the correct scope and hence BatchNorm was not being applied. Using the scope given [here](inception_v3_arg_scope), the problem is now resolved.

**Comment by LeeKyungMoon** [comment]
Thanks for your issues. But I'm confused about how you resolved this issue.

I'm aware of that biases could be removed because of Batch-Norm.

But I don't understand what you mean in "These were present in the model I created because I wasn't using the correct scope and ..." and the link you added is NOT_FOUND.

Thanks.

**Comment by cdac-cvml** [comment]
I think you are not passing model name argument properly. it should be passed as 
--model_name=inception_v3

**Comment by amina11** [comment]
@leekyungmoon  maybe it is obvious but I am confused why the biases disappear when we apply batchnorm?
